tests/test_s2_models.py

```diff
@@ -1,31 +0,0 @@
-from flexmeasures_client.s2.utils import get_unique_id
-from flexmeasures_client.s2.wrapper import S2Wrapper
-
-
-def test_simple_model():
-    wrapped_message = {
-        "message": {
-            "message_id": get_unique_id(),
-            "resource_id": get_unique_id(),
-            "roles": [{"role": "ENERGY_STORAGE", "commodity": "ELECTRICITY"}],
-            "instruction_processing_delay": 1.0,
-            "available_control_types": ["FILL_RATE_BASED_CONTROL", "NO_SELECTION"],
-            "provides_forecast": True,
-            "provides_power_measurement_types": ["ELECTRIC.POWER.3_PHASE_SYMMETRIC"],
-            "message_type": "ResourceManagerDetails",
-        },
-        "metadata": {"dt": "2023-01-01T00:00:00"},
```

Comment:
In my opinion, these should be forced to be timezone aware datetimes (and following the ISO 8601 standard), e.g. `"2023-01-01T00:00:00+00"`.